### DISCUSSION TITLE
[i18n] Configuration Options

### DISCUSSION THREAD

**Discussion by utsav0209** [vercel/next.js#31310] [Help]
Is there any way to use the configuration options suggested [here](https://www.i18next.com/overview/configuration-options) by i18-next with `next-i18next`? I tried passing options inside i18n config as shown below but could not get them working.

```javascript
module.exports = {
	i18n: {
		locales       : ['en', 'fr'],
		defaultLocale : 'en',
	},
	localePath: './src/public/locales',
        // i18n configs
	nonExplicitSupportedLngs: true,
	fallbackLng: {
	        "fr-US": ["fr"],
	}
};
```

So far have not been able to get this working, wondering if this is even possible to do?

**Comment by wiiiimm** [comment]
@utsav0209 which part of your i18n configuration is not working?

**Comment by utsav0209** [comment]
@williamli  while `fallbackLng:` works for me, I am not being able to make use of this flag: `nonExplicitSupportedLngs: true,` for some reason
